.github/workflows/inspektor-gadget.yml

```diff
@@ -395,7 +395,7 @@ jobs:
     - name: Build example binaries
       run: |
         for dir in $(dirname $(find examples/gadgets -name README.md)); do
-          go build ./$dir;
+          go build -tags withebpf ./$dir;
```

Comment:
I wonder if we could enable the usage of IG's packages in third-party software without requiting a change of tags in their build system.

Reply:
After internal discussion: We'll use `withoutebpf` instead of `withebpf`, so that the default is to build with the ebpf objects. For (the rarer) cases, where a third party explicitly doesn't want to build the ebpf objects, the `withoutebpf` build tag is required.

Reply:
There are still few places where this needs to be updated -> https://github.com/inspektor-gadget/inspektor-gadget/commit/607dd17b5cff9282a989422ccf8da4b6b525911d